coldfront/core/group/tests/management/test_create_default_user_groups.py

```diff
@@ -11,22 +11,43 @@ class CreateDefaultUserGroupsTest(TestCase):
     Test the create_default_user_groups management command
     """
 
-    def test_create_default_user_groups(self):
-        # Ensure no groups exist before running the command
-        self.assertEqual(Group.objects.count(), 0)
+    # POSITIVE TESTS
+    # Test that the command creates the RIS-UserSupport group
+    def test_ris_usersupport_group_created(self):
+        call_command("create_default_user_groups")
+        group = Group.objects.filter(name="RIS-UserSupport").first()
+        self.assertIsNotNone(group)
 
-        # Call the management command
+    # Test that the RIS-UserSupport group has all the permissions in the DEFAULT_RIS_USERSUPPORT_GROUP_PERMISSIONS list
+    def test_ris_usersupport_group_expected_permissions(self):
         call_command("create_default_user_groups")
+        group = Group.objects.filter(name="RIS-UserSupport").first()
+        group_permissions = group.permissions.all()
+        expected_permissions = Permission.objects.filter(
+            id__in=[
+                permission["id"]
+                for permission in DEFAULT_RIS_USERSUPPORT_GROUP_PERMISSIONS
+            ]
+        ).all()
+        self.assertEqual(
+            set(group_permissions),
+            set(expected_permissions),
+        )
 
-        # Check that the groups were created
-        for user_group in DEFAULT_RIS_USER_GROUPS:
-            new_user_group = Group.objects.get(name=user_group["name"])
-            self.assertIsNotNone(new_user_group)
-            self.assertEqual(
-                new_user_group.permissions.count(), len(user_group["permissions"])
-            )
+    # NEGATIVE TESTS
+    # Test that the RIS-UserSupport group has all permissions in the auth_permission table
```

Comment:
I'm guessing the comment should read "does not have all" instead of "has all"?

Reply:
Yeah I guess that is the case. My thinking was that I was testing a negative so it should evaluate to False. But on another read it seems you are correct.